### PR TITLE
Ci/add snap ci

### PR DESCRIPTION
**Pull request by Guillaumebeuzeboc**

### CODE REVIEW

**Comment by Guillaumebeuzeboc** [comment]
Secret has been added.

File: .github/workflows/monthly.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: Monthly
+on:
+  schedule:
+    - cron: '0 0 1 * *'
+  workflow_dispatch:
+
+jobs:
+  main:
+    uses: canonical/ros-foxglove-bridge/.github/workflows/snap.yaml@main
```
Comment (giusebar) [inline .github/workflows/monthly.yaml]: replace canonical with ubuntu-robotics

File: .github/workflows/snap.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+name: snap
+on:
+  push:
+    branches:
+      - feature/snap
```
Comment (giusebar) [inline .github/workflows/snap.yaml:5]: I see there is an old PR not merged from jeremie, do we keep the snap on feature/snap or should we merge everything to main?

**Review by giusebar — CHANGES_REQUESTED** [review]

File: .github/workflows/snap.yaml (hunk 2 above)
Comment (Guillaumebeuzeboc) [reply .github/workflows/snap.yaml:5]: They never replied about the snapping of the agent. I think if we want to keep it, we will simply have a standalone repository just for the snap.

**Review by giusebar — APPROVED** [review]

**Pull request merged**